atom/browser/api/atom_api_web_contents.h

```diff
@@ -181,6 +181,11 @@ class WebContents : public mate::TrackableObject<WebContents>,
                       const std::string& channel,
                       const base::ListValue& args);
 
+  bool SendIPCMessageWithSender(bool all_frames,
+                                const std::string& channel,
+                                const base::ListValue& args,
+                                int32_t sender_id = 0);
```

Comment:
Why not add this default fourth arg to `SendIPCMessage()` instead of adding a new function -- do we need both functions?

Reply:
It does not work for methods exposed to JS via mate. The default arguments are ignored. It has to be a separate method unfortunatelly